apps/dolly-frontend/src/main/js/src/components/fagsystem/pensjon/visning/PensjonVisning.js

```diff
@@ -35,23 +54,31 @@ export const PensjonVisning = ({ data, loading }) => {
 			/>
 			{manglerFagsystemdata ? (
 				<Alert variant={'warning'} size={'small'} inline style={{ marginBottom: '20px' }}>
-					Kunne ikke hente arbeidsforhold-data på person
+					Kunne ikke hente pensjon-data på person
 				</Alert>
 			) : (
-				<Panel
-					startOpen={runningTestcafe()}
-					heading={`Pensjonsgivende inntekter (${foerste} - ${siste})`}
-				>
-					<DollyFieldArray data={data.inntekter} nested>
-						{(inntekt, idx) => (
-							<div className="person-visning_content" key={idx}>
-								<TitleValue title="Inntektsår" value={inntekt.inntektAar} />
-								<TitleValue title="Beløp" value={inntekt.belop} />
-							</div>
-						)}
-					</DollyFieldArray>
+				<Panel startOpen={runningTestcafe()} heading={getTittel(data)}>
+					<PensjonInntekt data={data} />
 				</Panel>
 			)}
+			<PoppMiljoeinfo ident={data?.fnr} bestilteMiljoer={bestilteMiljoer} />
 		</ErrorBoundary>
 	)
 }
+
+export const PensjonvisningMiljo = ({ data, miljoe }) => {
+	const pensjonMiljoeInfo = data.find((m) => m.miljo == miljoe.toString())
+
+	return !pensjonMiljoeInfo ||
+		!pensjonMiljoeInfo?.inntekter ||
```

Comment:
Disse to sjekkene trengs vel ikke, dersom noe før length er undefined så går den til div med bare `pensjonMiljoeInfo?.inntekter?.length < 1` uansett 🤓

Reply:
Haha, veldig sant! 🤓